internal/config/network.go

```diff
@@ -42,6 +48,9 @@ var TestNet3 = &Network{
 	// DCP0005Height on testnet3 is block
 	// 0000003e54421d585f4a609393a8694509af98f62b8449f245b09fe1389f8f77.
 	DCP0005Height: 323328,
+	// DCP0010Height on testnet3 is block
+	// 00000000c78c04b8b6f7a6a4e3c602abff55f93924e154cac7be3a039941c6cd.
+	DCP0010Height: 867648,
```

Comment:
This isn't correct.

```sh
$ dcrctl --testnet getblockchaininfo | jq -c '.deployments.changesubsidysplit | {status,since}'
{"status":"active","since":877728}
$ dcrctl --testnet getblockchaininfo | jq .deployments.changesubsidysplit.since | dcrctl --testnet getblockhash -
000000000000c7fd75f2234bbff6bb81de3a9ebbd2fdd383ae3dbc6205ffe4ff
```

Reply:
Damn, looks like https://voting.decred.org/testnet is wrong. dcrdata matches the values you have.